ydb/mvp/oidc_proxy/oidc_protected_page.h

```diff
@@ -60,21 +60,19 @@ class THandlerSessionServiceCheck : public NActors::TActorBootstrapped<THandlerS
         TStringBuf authHeader = headers.Get(AUTH_HEADER_NAME);
         if (Request->Method == "OPTIONS" || IsAuthorizedRequest(authHeader)) {
```

Comment:
```
if (... IsAuthorizedRequest(authHeader) ...) {
    ForwardUserRequest(TString(authHeader), ctx, secure = false);
}
```

As I see, this is passing a request with Authorization header with secret token through unsecure environment. So it opens us to attack man in the moddle. It it possible to always send request through secure channel, if we are dealing with production environment?

Reply:
It depends on how the balancer is configured on meta. For production clusters, it is always configured with https - I think this should be enough to ensure the use of a secure channel.